R/translation.R

```diff
@@ -0,0 +1,21 @@
+# templated warning/error functions to smooth translation & development
+
+catf = function(fmt, ..., sep=" ", domain=NULL) {
```

Comment:
WDYT about adding new param `nl` that stands for "new line" and adds `\n` at the end of the string? IMO could be even by default true.

Reply:
Sounds reasonable... I wonder if it's mistake-prone though (tracking whether "\n" is done twice). I guess we're pretty used to writing a trailing \n with cat (and Rprintf for that matter)